nss-day-cohort-45/hello-world-column-counts
# Caa greece

Added links and photo, implemented CSS styling.

How to test: 
1) make sure CSS is properly formatted on my page
2) make sure links work
3) check that proper page is underlined in nav bar
